bootstrap.sh

```diff
@@ -7,4 +7,21 @@ mkdir -p $GRAPHITE_ROOT/storage/whisper
 
 python $GRAPHITE_ROOT/webapp/graphite/manage.py syncdb --noinput
 
+render_template() {
+    eval "echo \"$(cat $1)\""
+}
+
+ENABLE_AMQP=${ENABLE_AMQP:-False}
+AMQP_VERBOSE=${AMQP_VERBOSE:-False}
+
+AMQP_HOST=${AMQP_HOST:-localhost}
```

Comment:
Please make these quoted, e.g. `AMQP_HOST="${AMQP_HOST:-localhost}"` so that things like passwords are a bit more special symbol-proof.